Application/EdFi.Ods.AdminApi/Infrastructure/OdsSecurityVersionResolver.cs

```diff
@@ -39,7 +38,7 @@ private EdFiOdsSecurityModelCompatibility InitializeModelVersion()
         if (!validationResult.IsValidOdsApi || validationResult.Version == null)
         {
             _log.Error("Unable to determine security model from ODS API Response");
-            throw validationResult.Exception ?? new Exception("No version reported from ODS API");
+            throw validationResult.Exception ?? new Exception("No version reported from the Ed-Fi API");
```

Comment:
I changed that thinking about a generic "Ed-Fi API", not just our specific "Ed-Fi ODS/API". However, now that I think about it more, it was just fine before: the Admin API is really intended only for support of the Ed-Fi ODS/API. Either way is fine though, so I won't change it back right now.